.github/workflows/docker-build.yml

```diff
@@ -104,7 +103,6 @@ jobs:
         include:
           # These are the base images for PL release docker images,
           # so include at least all of the combinations in release-dockers.yml.
```

Comment:
Did you update rlease-dockers.yml as this comment suggests?

Reply:
@Borda must have renamed it at some point. Any suggestions Jirka?

Reply:
docker workflows were merged into one: https://github.com/Lightning-AI/pytorch-lightning/blob/master/.github/workflows/docker-build.yml

Reply:
Ok, since they were combined into the same image, I updated the comment. 

While we are here, I think L77 should also be changed to tag the latest correctly?